Main.java

```diff
@@ -1,3 +1,4 @@
+import java.util.Scanner;
 /**
  *
  * @author 

```

Comment:
Don't forget the code header. It should contain a description what the program does and also your name.